### PR TITLE
List in columns

### PR DESCRIPTION
**Pull request by fondasimone**

### CODE REVIEW

File: pride/src/main/java/com/prezi/pride/cli/commands/ListCommand.java [hunk 1]
```diff
@@ -68,12 +85,17 @@ public String formatModule(Module module) throws Exception {
 			VcsStatus status = module.getVcs().getSupport().getStatus(moduleDirectory);
 			String branch = status.getBranch();
 			StringBuilder line = new StringBuilder();
+
 			line.append(status.hasUnpublishedChanges() ? 'M' : ' ');
 			line.append(status.hasUncommittedChanges() ? 'M' : ' ');
 			line.append(' ').append(module.getName());
-			line.append(' ');
+
+			line.append(new String(new char[moduleMaxLength + 2 - module.getName().length()]).replace("\0", " "));
```
Comment (lptr) [inline pride/src/main/java/com/prezi/pride/cli/commands/ListCommand.java]: Whoa, this is an original idea. :) But not very efficient. Please use `Strings.repeat()` instead.

Comment (fondasimone) [inline pride/src/main/java/com/prezi/pride/cli/commands/ListCommand.java]: :police_car: :police_car: :police_car: !

Not my invention though.. i just stackoverflow-ed how to repeat a string in java and picked the one which looked coolest :D 

I'll replace it with Strings.repeat().

**Comment by fondasimone** [comment]
Ping @lptr @vujevits

File: pride/src/main/java/com/prezi/pride/cli/commands/ListCommand.java [hunk 2]
```diff
@@ -56,9 +56,22 @@ public String formatModule(Module module) {
 	private static class StatusFormatter implements LineFormatter {
 
 		private final Pride pride;
+		private int moduleMaxLength = 0;
+		private int branchMaxLength = 0;
 
-		private StatusFormatter(Pride pride) {
+		private StatusFormatter(Pride pride) throws Exception {
```
Comment (lptr) [inline pride/src/main/java/com/prezi/pride/cli/commands/ListCommand.java]: This should be `throws IOException`, unless any other exception is thrown here.

File: pride/src/main/java/com/prezi/pride/cli/commands/ListCommand.java [hunk 3]
```diff
@@ -56,9 +56,22 @@ public String formatModule(Module module) {
 	private static class StatusFormatter implements LineFormatter {
 
 		private final Pride pride;
+		private int moduleMaxLength = 0;
+		private int branchMaxLength = 0;
```
Comment (lptr) [inline pride/src/main/java/com/prezi/pride/cli/commands/ListCommand.java]: You don't need to set these to `0` explicitly -- that's the default value,

File: pride/src/main/java/com/prezi/pride/cli/commands/ListCommand.java (hunk 2 above)
Comment (fondasimone) [inline pride/src/main/java/com/prezi/pride/cli/commands/ListCommand.java]: Then why: public String formatModule(Module module) throws Exception ?

**Comment by lptr** [comment]
Now the build is broken. Please import `IOException`.

**Comment by fondasimone** [comment]
Darn, im so bad.  Fixed, thanks.

Btw, if you made me throwing IOException, why is #formatModule throwing Exception?

**Comment by lptr** [comment]
I guess that should throw `IOException` as well. I'm going to look into it.

**Pull request merged**